### ISSUE TITLE
docs/overview.md: dynamicFmtString missing from checkers documentation.

### ISSUE THREAD

**Issue by ErgoAsciiCat** [go-critic/go-critic#1270]
DynamicFmtString is a functional checker that is reported via `gocritic doc` and has been linting successfully.

`gocritic doc dynamicFmtString` returns:
```
dynamicFmtString checker documentation
URL: https://github.com/go-critic/go-critic/checkers
Tags: [diagnostic experimental]

Detects suspicious formatting strings usage.

Non-compliant code:
fmt.Errorf(msg)

Compliant code:
fmt.Errorf("%s", msg)
```

It does not have an entry in [checkers](https://go-critic.com/overview) and lists an old URL. PTAL

**Comment by cristaloleg** [comment]
@quasilyte it's not visible 'cause it's implemented as a ruleguard rule?

**Comment by quasilyte** [comment]
It should still be visible in the documentation.
I think we should re-generate and upload the new docs.
